package.json

```diff
@@ -74,7 +74,7 @@
     "prestart": "npm run build",
     "start": "source env/bin/activate || . env/bin/activate && python run.py",
     "start:test": "npm run build && source env/bin/activate || . env/bin/activate && python run.py -t",
-    "virtualenv": "pip install virtualenv && virtualenv -p python env && source env/bin/activate || . env/bin/activate && npm run pip",
+    "virtualenv": "python3 -m venv env && source env/bin/activate || . env/bin/activate && npm run pip",
     "pip": "pip install -r config/requirements-dev.txt",
```

Comment:
Do we want to point to `pip3` here? Or just keep as pip?

We need to update the docs in the detailed section as well, because they recommend `pip2`: https://github.com/webcompat/webcompat.com/blob/e1609766663673f56d547240dd7a893adec45fcc/docs/dev-env-setup.md#installing-pip

Reply:
so once the `virtualenv` has been started the version of pip which is available is pip3.

```
→ which pip
/Users/karl/code/webcompat.com/env/bin/pip
→ pip -V
pip 19.2.2 from /Users/karl/code/webcompat.com/env/lib/python3.7/site-packages/pip (python 3.7)
```

but yes the detailed version needs to be updated too.

Reply:
updated the detailed section.